frontend/app/model/Eventbrite.scala

```diff
@@ -12,6 +12,7 @@ import play.api.data.validation.ValidationError
 import play.api.libs.functional.syntax._
 import play.api.libs.json._
 import utils.StringUtils._
+import views.support.Asset
```

Comment:
Considering moving this out of views support as we use it in a few places outside views now…